File: katas/7-kyu/tram-capacity/solution.py
def tram(stops, descending, onboarding):
    curr_pas = 0;
    min_cap = 0;
    for i in range(stops):
        curr_pas-= descending[i]
        curr_pas+= onboarding[i]
        min_cap = max(curr_pas, min_cap)
    return min_cap

File: katas/7-kyu/tram-capacity/solution.js
function tram(stops, descending, onboarding){
  //your code here
  let minCap = 0;
  let currPas = 0;
  for (let i = 0; i < stops; i++){
    currPas-= descending[i];
    currPas+= onboarding[i];
    minCap = Math.max(minCap, currPas)
  }
  
  return minCap;
}